test/parallel/test-runner-run.mjs

```diff
@@ -4,6 +4,7 @@ import { join } from 'node:path';
 import { describe, it, run } from 'node:test';
 import { dot, spec, tap } from 'node:test/reporters';
 import assert from 'node:assert';
+import utils from 'internal/util/inspect';
```

Comment:
I believe this needs the following at the top for the new `internal` import to work:

```js
// Flags: --expose-internals
```

Reply:
yeah I just figured it out, I should have read this comment earlier 😄 I thought it was there in other tests just as a reference, I didn't know it was actually functional...

Reply:
You could also use the function exported from the public API.

Reply:
yeah even using the `Flags` comment didn't work, let's try with the public API this time